frontend/src/lib/util/urls.ts

```diff
@@ -0,0 +1,19 @@
+import type { ConditionalPick } from 'type-fest';
+import type { PrimitiveRecord } from './types';
+
+// https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1568#issuecomment-1587963141
```

Comment:
this is nice, but can you explain what the link is explaining, I had to click the link to find out why you included one in the first place.